src/components/reusable_components/CustomSearch.js

```diff
@@ -69,42 +66,42 @@ export default function CustomSearch(props) {
         }}
       />
       {/* Render the list of results after loading is done */}
-      {searchResults && !isLoading ? (
+      {searchResults && !isLoading && searchQuery !== "" ? (
         <FlashList
           data={searchResults}
-          keyExtractor={(item) => {
-            return item.id;
-          }}
+          keyExtractor={(item, index) => index.toString()}
           viewabilityConfig={{
             waitForInteraction: true,
             itemVisiblePercentThreshold: 100,
             minimumViewTime: 1000,
           }}
-          estimatedItemSize={mockData.length}
-          onEndReached={() => console.log("Reached bottom")}
+          estimatedItemSize={searchResults.length}
           renderItem={({ item }) => (
             <TouchableOpacity
               style={styles.resultListItem}
               onPress={() =>
-                searchType === "standard"
+                searchType === "standard" &&
+                !searchResults.includes("Ticker not found")
                   ? navigation.navigate("StockInfoScreen", {
-                      stockName: item.name,
+                      stockName: item,
                     })
                   : null
               }
             >
               <View style={styles.textCell}>
-                <Text style={styles.text}>{item.abbreviation}</Text>
+                <Text style={styles.text}>{item}</Text>
               </View>
               <View style={styles.textCell}>
-                <Text style={styles.text}>{item.name}</Text>
+                <Text style={styles.text}>{item}</Text>
               </View>
             </TouchableOpacity>
           )}
         />
       ) : (
-        <View style={styles.activityIndicator}>
-          <ActivityIndicator size="large" color={THEME.colors.primary} />
+        <View>
+          <Text style={styles.text2}>
```

Comment:
Can we get rid of this?

Reply:
yea we can get rid of the text, there just has to be something there because the flashlist needs at least one element in there. So that shows when the flashlist is empty or there is no results

Reply:
done